homematic/Dockerfile

```diff
@@ -75,6 +75,8 @@ RUN \
     && touch /opt/hm/etc/config/firewallConfigured \
     && touch /opt/hm/etc/config/legacyAPIMigrationAccepted \
     \
+    && ln -s /data/groups.gson /opt/hm/etc/config/groups.gson \
```

Comment:
Please use `https://github.com/home-assistant/addons/blob/master/homematic/rootfs/etc/cont-init.d/env.sh` for this.